libavcodec/x86/vvc/vvcdsp_init.c

```diff
@@ -253,16 +253,14 @@ AVG_FUNCS(16, 12, avx2)
     c->inter.w_avg  = bf(ff_vvc_w_avg, bd, opt);                     \
 } while (0)
 
-int ff_vvc_sad_8_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
-int ff_vvc_sad_16_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
-int ff_vvc_sad_32_128_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
-
-#define SAD_INIT() do {                                           \
-    c->inter.sad[1] = ff_vvc_sad_8_avx2;                          \
-    c->inter.sad[2] = ff_vvc_sad_16_avx2;                         \
-    c->inter.sad[3] = ff_vvc_sad_32_128_avx2;                     \
-    c->inter.sad[4] = ff_vvc_sad_32_128_avx2;                     \
-    c->inter.sad[5] = ff_vvc_sad_32_128_avx2;                     \
+int ff_vvc_sad_avx2(const int16_t *src0, const int16_t *src1, int dx, int dy, int block_w, int block_h);
+
+#define SAD_INIT() do {                                    \
+    c->inter.sad[1] = ff_vvc_sad_avx2;                     \
```

Comment:
a loop may be better

Reply:
or you do not need change to sad[], just use sad since you only have one fuction

Reply:
yeah that's the plan is to just remove it completely